components/Table/style.scss

```diff
@@ -46,44 +46,46 @@
   }
 }
 
-.markdown-body { 
+.markdown-body {
   @include markdown-table;
+
   .rdmd-table {
     $border-wrap-width: 1px;
+
     & {
-      position: relative;
       display: block;
+      position: relative;
     }
-    &:after {
-      content: ' ';
-      position: absolute;
-      box-shadow: inset 0 0 0 $border-wrap-width var(--table-edges, #dfe2e5);
-      width: 100%;
-      height: 100%;
-      top: 0;
-      pointer-events: none;
-      z-index: 1;
-    }
+
     &-inner {
-      min-width: 100%;
       box-sizing: content-box;
+      min-width: 100%;
+      overflow: auto;
       width: 100%;
     }
-    table:only-child {
-      margin: 0 !important;
-      thead th {
-        position: sticky;
-        left: 0;
-        z-index: 1;
-        background: inherit;
-      }
-      td:last-child,
-      th:last-child {
-        border-right: none;
-      }
-      thead tr,
-      thead th:last-child {
-        box-shadow: 3px 0 0 var(--table-head);
+
+    table {
+      border: 1px solid var(--table-edges, #dfe2e5);
+
+      &:only-child {
+        margin: 0 !important;
+
+        thead th {
+          background: inherit;
+          left: 0;
+          position: sticky;
+          z-index: 1;
```

Comment:
Should we just get rid of this? I did it originally, but now I feel it looks kinda janky most of the time?

Reply:
I'm down! Yeah, it looks a little janky sometimes with all the scrolling now. 👍

Reply:
Made a suggested edit for it! 👍

https://github.com/readmeio/markdown/pull/160/files/ba6a949d701a659966ac3093ea25fd5f55c4a8ae#r637438330